update codewowners

cristinabuenahora opened the pull request:
## Change description

Removed @nikhilunni from codeowners + added Igor & Jason 

Question -- Do we still want to use codeowners?

FoodProduct commented:
> Question -- Do we still want to use codeowners?

Let's go back to first principles. What is the goal of codeowners here/what problem is its usage intended to solve?

cristinabuenahora commented:
> > Question -- Do we still want to use codeowners?
> 
> Let's go back to first principles. What is the goal of codeowners here/what problem is its usage intended to solve?

Back in the day, we implemented it to auto tag reviewers who own this codebase.

cristinabuenahora commented:
> > > Question -- Do we still want to use codeowners?
> > 
> > 
> > Let's go back to first principles. What is the goal of codeowners here/what problem is its usage intended to solve?
> 
> Back in the day, we implemented it to auto tag reviewers who own this codebase.



> > > Question -- Do we still want to use codeowners?
> > 
> > 
> > Let's go back to first principles. What is the goal of codeowners here/what problem is its usage intended to solve?
> 
> Back in the day, we implemented it to auto tag reviewers who own this codebase.

Merging this for now & let's discuss codeowners as part of the conversation around longterm support for Backstage.
